### PR TITLE
Use ascii fields for UTF-8 search terms in Page

### PR DESCRIPTION
**Pull request by bricesanchez**
We are not able to search query with UTF-8 characters, this PR fix it.

### CODE REVIEW

File: spec/features/refinery/acts_as_indexed/admin/search_spec.rb [hunk 1]
```diff
@@ -46,14 +46,14 @@ module Refinery
...
     describe "pages extension" do
       before do
-        FactoryGirl.create(:page, :title => "Ugis Ozols")
+        FactoryGirl.create(:page, :title => "Téléphone")
         visit refinery.admin_pages_path
       end
 
       it "returns found page" do
-        fill_in "search", :with => "ugis"
+        fill_in "search", :with => "Telephone"
```
Comment (simi) [inline spec/features/refinery/acts_as_indexed/admin/search_spec.rb]: I think you should search here for non-ascii variant.

Comment (bricesanchez) [inline spec/features/refinery/acts_as_indexed/admin/search_spec.rb]: I could make 2 searchs, with and without ascii characters

**Comment by bricesanchez** [comment]
:up: ?

**Comment by parndt** [comment]
@bricesanchez is this still required / desired?

**Comment by bricesanchez** [comment]
As i remember, `acts_as_indexed` still have problems with accents in UTF-8, it was a good fix, i will have to rebase it to see if it always works.

**Comment by parndt** [comment]
@bricesanchez looking forward to confirmation

File: README.md [hunk 2]
```diff
@@ -14,11 +14,20 @@ Refinery CMS Acts As Indexed is the [Acts As Indexed](https://github.com/dougal/
...
 ```ruby
-gem 'refinerycms-acts-as-indexed', '~> 2.0.1'
+gem 'refinerycms-acts-as-indexed', '~> 3.0.0'
 ```
 
 Now run `bundle install`.
 
+## Upgrade
+
+When upgrading to a new version of `refinerycms-acts-as-indexed` it is recommended you delete the index directory and allow it to be rebuilt.
```
Comment (parndt) [inline README.md]: How?

Comment (bricesanchez) [inline README.md]: Good question!

Comment (parndt) [inline README.md]: Actually that's the advice given by the main repo, so all good.

**Pull request merged**